pages/api/product/[id].js

```diff
@@ -1,9 +1,11 @@
 import nc from 'next-connect';
 import productService from '../../../src/server/services/product.service';
 import validationSchema from '../../../src/server/validations/products.validation';
+import middlewareLogs from '../../../src/server/middlewares/logger.middleware';
```

Comment:
rename imports from `middlewareLogs ` to `loggerMiddleware` as it is declared in a middleware file.